libs/schematic/api-description/src/openapi.yaml

```diff
@@ -106,3 +106,6 @@ paths:
 
   /tangledTreeText:
     $ref: paths/tangledTreeText.yaml
+
```

Comment:
For generating an existing manifest, user would need to put in access token like other endpoints (because schematic would need to pull manifests from a dataset ID provided by the user). Could you add a way for user to add their synapse token like other APIs?

Reply:
Yep, I must have forgotten to add that!